### ISSUE TITLE
BAB/CMB/CMD parsing broken

### ISSUE THREAD

**Issue by Lavaeolous** [Lavaeolous/PF1-StatBlock-Converter-Module#598]

**Comment by Lavaeolous** [comment]
e.g.
https://www.aonprd.com/NPCDisplay.aspx?ItemName=Fierani%20Demon%20Hunter
https://www.aonprd.com/NPCDisplay.aspx?ItemName=Swampwalker

with for example
```Base Atk +12; CMB +13; CMD 29
Base Atk +9; CMB +13; CMD 25```
getting parsed as
```Base Atk +1; CMB +2; CMD 19
Base Atk +1; CMB +5; CMD 16```

There may be general parsing errors (truncating the string or something like that) or calculation errors.

**Comment by Lavaeolous** [comment]
This seems to relate to #599. In general i should transition to just using these values for validation, as the totals are derived and overwritten by foundry and the pf1 system anyway.

**Comment by Lavaeolous** [comment]
Fixed in 4.0.1, general idea of using the values only for validation added to backlog
